requirements.txt

```diff
@@ -1,2 +1,2 @@
 argparse; python_version < '2.7'
-# ovirt-engine-sdk >= 3.6  # It is not available in pip repo
+ovirt-engine-sdk-python == 3.6.9.2
```

Comment:
That's not needed - v4 is called differently (sdk4).
There is not newer ovirt-engine-sdk-python planned to be released in 3.6.x.